src/Helpers/JsonDecoder.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
These two classes are available in https://packagist.org/packages/digiaonline/json-helpers nowadays